specification/07-theory_operations.adoc

```diff
@@ -360,73 +424,89 @@ The IDE link initial setup must go through the following steps:
         message is encapsulated into a vendor-defined SPDM request.
     b. Receives an `IDE_KM_KEY K_GOSTOP_ACK` from the DSM.
 
-
-.Device Connection
+.Device Connection - IDE Link Setup
 [source,mermaid]
 ....
 %%{init: {'theme': 'neutral', 'themeVariables': {'darkMode': true}, "flowchart" : { "curve" : "basis" } } }%%
 sequenceDiagram
 
 autonumber
 
-participant DSM as Device DSM or ROT
+participant DSM as Device DSM
+participant Device as PCIe Device
+participant RootPort as PCIe Root Port
+participant RoT as RoT (RP DSM)
 participant VMM as Host Supervisor Domain Manager (VMM)
 participant TSM
 
-VMM ->> TSM: [COVH] - sbi_covh_connect_device()
-
-Note over TSM,DSM: TSM: SPDM Connection Setup
-
-TSM ->> TSM: Generate SPDM request: GET_VERSION
-TSM ->> VMM: [COVH] - spdm_req(GET_VERSION)
-VMM ->> DSM: [DOE] - SPDM_GET_VERSION
-DSM ->> VMM: [DOE] - SPDM_VERSION
-VMM ->> TSM: [COVH] - spdm_resp(VERSION)
-
-TSM ->> TSM: Generate SPDM request: GET_CAPABILITIES
-TSM ->> VMM: [COVH] - spdm_req(GET_CAPABILITIES)
-VMM ->> DSM: [DOE] - SPDM_GET_CAPABILITIES
-DSM ->> VMM: [DOE] - CAPABILITIES
-VMM ->> TSM: [COVH] - spdm_resp(CAPABILITIES)
-
-TSM ->> TSM: Generate SPDM request: NEGOTIATE_ALGORITHMS
-TSM ->> VMM: [COVH] - spdm_req(NEGOTIATE_ALGORITHMS)
-VMM ->> DSM: [DOE] - SPDM_NEGOTIATE_ALGORITHMS
-DSM ->> VMM: [DOE] - ALGORITHMS
-VMM ->> TSM: [COVH] - spdm_resp(ALGORITHMS)
-
-TSM ->> TSM: Generate SPDM request: GET_CERTIFICATE
-TSM ->> VMM: [COVH] - spdm_req(GET_CERTIFICATE)
-VMM ->> DSM: [DOE] - GET_CERTIFICATE
-DSM ->> VMM: [DOE] - CERTIFICATE
-VMM ->> TSM: [COVH] - spdm_resp(CERTIFICATE)
-TSM ->> TSM: Verify and store device certificate chain
-
-Note over TSM,DSM: TSM: SPDM Key Exchange
-
-TSM ->> TSM: Generate ephemeral SPDM keys K0
-TSM ->> TSM: Generate SPDM request: KEY_EXCHANGE(K0)
-TSM ->> VMM: [COVH] - spdm_req(KEY_EXCHANGE)
-VMM ->> DSM: [DOE] - SPDM_KEY_EXCHANGE
-DSM ->> VMM: [DOE] - KEY_EXCHANGE_RSP
-VMM ->> TSM: [COVH] - spdm_resp(KEY_EXCHANGE_RSP)
-TSM ->> TSM: Derive DHE secret
-TSM ->> TSM: Generate SPDM request: FINISH
-TSM ->> VMM: [COVH] - spdm_req(FINISH)
-VMM ->> DSM: [DOE] - SPDM_FINISH
-DSM ->> VMM: [DOE] - FINISH_RSP
-VMM ->> TSM: [COVH] - spdm_resp(FINISH_RSP)
-
-Note over TSM,DSM: TSM: SPDM Session Created
-
-%% TODO Add the IDE sequence...
-Note over TSM,DSM: IDE Link setup starts here...
+VMM ->> VMM: Find an available IDE stream ID
+VMM ->> Device: Program IDE extended capability
```

Comment:
I think we should be explicit to say: "Ensure IDE is disabled"

Then after key is programmed, then we say "Enable IDE.". See comment later.

Reply:
Done. I expanded the VMM step to have it explicitly disable IDE. Obviously this would be a NOP if it is already disabled.